bodhi/server/templates/update.html

```diff
@@ -785,7 +785,7 @@ <h5 class="m-t-2">
                     </tr>
                     % endif
 
-                    % if update.days_to_stable and update.status.description == 'testing':
+                    % if update.days_to_stable > 0 and update.status.description == 'testing':
```

Comment:
I think the bug is actually in the calculation of ```days_to_stable```. That code lives [here](https://github.com/fedora-infra/bodhi/blob/28dd380a1d077de3953a9464d765cea9e6db1235/bodhi/server/models.py#L1965-L1977):

```
        if not self.meets_testing_requirements and self.date_testing:
            return (
                self.release.mandatory_days_in_testing -
                (datetime.utcnow() - self.date_testing).days)
        return 0
```

The issue seems to be that critical path updates have a different setting for how many days they should stay in testing, and this code doesn't take that into account. [```meets_testing_requirements```](https://github.com/fedora-infra/bodhi/blob/28dd380a1d077de3953a9464d765cea9e6db1235/bodhi/server/models.py#L1903-L1937) returns ```False``` for critical path updates (like the one I cited in the bug I filed), because it is using ```config.get('critpath.stable_after_days_without_negative_karma')``` to find the number of days required in testing.

So I think the thing to do is to create a new method on the Update model, perhaps ```Update.mandatory_days_in_testing()``` (analogous to the ```Release.mandatory_days_in_testing()``` method), that could look something like this:

```
if self.critpath:
    return int(config.get('critpath.stable_after_days_without_negative_karma'))

num_days = self.release.mandatory_days_in_testing
return num_days if num_days else 0
```

Then we could modify ```meets_testing_requirements``` and ```days_to_stable``` to both use this source of truth for their respective calculations. I found one other place in ```models.py``` that also uses the ```critpath.stable_after_days_without_negative_karma``` setting that could be updated to use this. There are also a few spots that use ```self.release.mandatory_days_in_testing()``` that could use the new ```self.mandatory_days_in_testing()```. How does that sound?